### DISCUSSION TITLE
[Feature]: E-mail activation of new account

### DISCUSSION THREAD

**Discussion by CyberFox001** [django-wiki/django-wiki#1266] [Ideas]
### I've already checked other feature proposals

Yes

### Is your feature request related to a problem?

Yes

### Describe your problem.

When you create a new account with the form, it's directly activated.

### Describe your feature request.

It would be nice to have an option that use e-mail to confirm account create.

When a new account is created, an e-mail is sent to the address given by the user. On this e-mail, there is an link to open in order to activate the account.

**Comment by oscarmcm** [comment]
Hi @CyberFox001 

Thank you for opening this discussion, I'm not fully sure if implementing this will be a good idea since there are a few alternatives that can accomplish this.

You can create your own authentication method or use a third party app like [Django-Registration](https://django-registration.readthedocs.io/en/3.0/activation-workflow.html) which provides the two-steps authentication, either way if you use a third party app or a custom, remember to set `settings.WIKI_ACCOUNT_HANDLING` to `False`.

**Comment by CyberFox001** [comment]
Hi @oscarmcm 

I tested Django-wiki with Django-Registration. It took me some time to create all the templates needed by Django registration and django.contrib.auth.

But I have two problems with the user menu in the wiki.

First, to configure the "Log out" entry to point to the new logout url, I had to set the variable `LOGOUT_URL` in the settings. This point is missing in the documentation. Can I help fixing it ?

Second, the entry "Account Settings" is disabled when the account handling is disabled. And I can't found a way to insert a new entry to point to the django.contrib.auth url to modify the password. Is account settings have to be disabled when account handling is ?
